integration-tests/scripts/gen-config.ts

```diff
@@ -0,0 +1,44 @@
+import * as Bun from "bun";
+import { unlink } from "node:fs/promises";
+
+interface CreateConfigOpts {
+  bundler: "rollup" | "vite" | "webpack";
+  format: string;
+  detectFormat: string;
+  version: string;
+  detectVersion: string;
+  file_format: "ts" | "cjs";
+}
+
+export async function createConfig({
+  bundler,
+  format,
+  detectFormat,
+  version,
+  detectVersion,
+  file_format,
+}: CreateConfigOpts) {
+  const bundlerDir = `fixtures/generate-bundle-stats/${bundler}`;
+  const baseConfigPath = `${bundlerDir}/${bundler}-base.config.${file_format}`;
+  const outFileName = `${bundler}-${version}-${format}.config.${file_format}`;
+  const outFilePath = `${bundlerDir}/${outFileName}`;
+
+  const file = Bun.file(baseConfigPath);
+
+  const outFile = Bun.file(outFilePath);
+  const outFileExists = await outFile.exists();
+
+  if (outFileExists) {
+    await unlink(outFilePath);
+  }
+
+  const upperCaseVersion = version.toUpperCase();
+  const upperCaseDetectVersion = detectVersion.toUpperCase();
+
+  let contents = await file.text();
+  contents = contents.replaceAll(detectFormat, format);
+  contents = contents.replaceAll(detectVersion, version);
+  contents = contents.replaceAll(upperCaseDetectVersion, upperCaseVersion);
```

Comment:
chaining the `replaceAll` calls cleans this up a little bit.

```suggestion
  const contents = await file.text()
    .replaceAll(detectFormat, format)
    .replaceAll(detectVersion, version)
    .replaceAll(upperCaseDetectVersion, upperCaseVersion);
```

Reply:
TS isn't happy with this because of the await, but I can do something like:

```ts
  const baseConfigContents = await file.text();
  const contents = baseConfigContents
    .replaceAll(detectFormat, format)
    .replaceAll(detectVersion, version)
    .replaceAll(upperCaseDetectVersion, upperCaseVersion);
```

Reply:
Ah yeah you'll have to do `(await file.text())`, but I find your version cleaner!